content/en/contribute/code/releasing.md

```diff
@@ -10,7 +10,34 @@ aliases: >
 
 ## CHT Core
 
-[Create an issue](https://github.com/medic/cht-core/issues/new/choose) for either a Major/Minor or Patch release issue template and assign it to the release manager to follow the process.
+A release is a set of code changes bundled together, ideally with at least one deployment of CHT apps ready to make use of it.
+
+### Building & Releasing CHT Core Changes
+
+The high-level steps for a release are as follows:
+
+* The [Focused Working Group]({{% ref "/contribute/medic/product-development-process/focused-groups" %}}) sees an opportunity they want to go after. The opportunity addresses a need of at least one CHT app deployment and will be used by that deployment after the release.
+* The Focused Working Group agrees on a solution for it.
```

Comment:
This content has a lot overlap with what we talk about on the [Feature Release page](https://docs.communityhealthtoolkit.org/core/releases/feature_releases/). Maybe we should consider merging the two pages or possibly moving the FR page to `content/en/contribute/code/*` so it's side by side?

Reply:
Great point @mrjones-plip, thanks for redirecting me to the FR page. I moved it under the `content/en/contribute/code/*` as it makes sense to keep it together with the release guidelines.